### PR TITLE
disable all action buttons when removing snapshot

### PR DESCRIPTION
**Pull request by bond95**
Fixes: https://github.com/oVirt/ovirt-web-ui/issues/792

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by sjd78** [comment]
How does it look now when an action is in progress as opposed to #792?

**Comment by bond95** [comment]
@sjd78 

- When you create snapshot you can delete one (for me it not delete snapshot, but don't return error as well, so it would be nice if you try it on your engine), but not restore snapshot
- When restore snapshot, you can't do anything.
- When you delete snapshot, you can only delete other ones.

Thus I think, if all actions will be disabled it would be more clear for user, then learn how actions interact with each other, and why some actions are enabled and some disabled.

**Comment by gregsheremeta** [comment]
https://jenkins.ovirt.org/job/oVirt_ovirt-web-ui_standard-check-pr/96/artifact/build-artifacts.el7.x86_64/ovirt-web-ui-1.5.0-0.20181011git9437c14.el7.noarch.rpm

https://jenkins.ovirt.org/job/oVirt_ovirt-web-ui_standard-check-pr/96/artifact/build-artifacts.fc28.x86_64/ovirt-web-ui-1.5.0-0.20181011git9437c14.fc28.noarch.rpm

File: src/components/VmDetails/cards/SnapshotsCard/index.js [hunk 1]
```diff
@@ -41,7 +43,7 @@ Snapshots.propTypes = {
 }
 
 const ConnectedSnapshots = connect(
-  (state) => ({
+  (state, { snapshots }) => ({
```
[the remark this answers is not in the document]
Comment (gregsheremeta) [reply src/components/VmDetails/cards/SnapshotsCard/index.js]: @bond95 can you post a fix for that?

**Comment by bond95** [comment]
@sjd78 could you, please, look at it again?

**Comment by gregsheremeta** [comment]
ci build please

**Comment by leistnerova** [comment]
LGTM

**Pull request merged**